deploy/crds/shipwright.io_buildruns.yaml

```diff
@@ -357,22 +313,12 @@ spec:
                     type: string
                 type: object
               latestTaskRunRef:
-                description: "LatestTaskRunRef is the name of the TaskRun responsible
-                  for executing this BuildRun. \n TODO: This should be called something
-                  like \"TaskRunName\""
-                type: string
-              reason:
```

Comment:
I have this one also here https://github.com/shipwright-io/build/pull/781/commits/273cca502dba3b2dfa2a23b1326039bfffa9dce6 .